tests/test_models/test_nn/test_chronos.py

```diff
@@ -0,0 +1,275 @@
+import os
+from pathlib import Path
+
+import pytest
+from pandas.testing import assert_frame_equal
+
+from etna.datasets import TSDataset
+from etna.datasets import generate_ar_df
+from etna.libs.chronos.chronos import ChronosModelForForecasting
+from etna.libs.chronos.chronos_bolt import ChronosBoltModelForForecasting
+from etna.models.nn import ChronosBoltModel
+from etna.models.nn import ChronosModel
+from etna.pipeline import Pipeline
+
+
+@pytest.fixture
+def ts_increasing_integers():
+    df = generate_ar_df(start_time="2001-01-01", periods=10, n_segments=2)
+    df["target"] = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9] + [100, 101, 102, 103, 104, 105, 106, 107, 108, 109]
+    ts = TSDataset(df, freq="D")
+    return ts
+
+
+@pytest.fixture
+def expected_ts_increasing_integers():
+    df = generate_ar_df(start_time="2001-01-11", periods=1, n_segments=2)
+    df["target"] = [10.0] + [110.0]
+    ts = TSDataset(df, freq="D")
+    return ts
+
+
+@pytest.mark.smoke
+def test_chronos_fail_unknown_model_name():
+    with pytest.raises(NotImplementedError, match="Model chronos-t5-supertiny is not available."):
+        _ = ChronosModel(model_name="chronos-t5-supertiny")
+
+
+@pytest.mark.smoke
+def test_chronos_bolt_fail_unknown_model_name():
+    with pytest.raises(NotImplementedError, match="Model chronos-bolt-supertiny is not available."):
+        _ = ChronosBoltModel(model_name="chronos-bolt-supertiny")
+
+
+@pytest.mark.smoke
+def test_chronos_custom_cache_dir(tmp_path):
+    _ = ChronosModel(model_name="chronos-t5-tiny", cache_dir=tmp_path)
+    assert os.path.exists(tmp_path)
+
+
+@pytest.mark.smoke
+def test_chronos_bolt_custom_cache_dir(tmp_path):
+    _ = ChronosBoltModel(model_name="chronos-bolt-tiny", cache_dir=tmp_path)
+    assert os.path.exists(tmp_path)
```

Comment:
As I understand correctly, this path will exist even without creating of chronos model.